hack/machineset.sh

```diff
@@ -75,16 +75,14 @@ EOF
 # get_aws_ms creates a MachineSet for AWS Cloud Provider
 get_aws_ms() {
 
-  if [ "$#" -lt 6 ]; then
+  if [ "$#" -lt 4 ]; then
     error-exit incorrect parameter count for get_aws_ms $#
   fi
 
   local infraID=$1
-  local region=$2
-  local az=$3
-  local provider=$4
-  local winver=$5
-  local byoh=$6
+  local linuxWorkerSpec=$2
+  local winver=$3
+  local byoh=$4
```

Comment:
I recommend moving the machineset.sh improvements to a separate PR as they are not related to WINC-1189,  making them easier to backport.

Reply:
I suppose you are right.
We need to backport WINC-1189, but we don't need to maintain this script in any branch but master.
I will break it out to a separate PR.